core/src/banking_stage/consumer.rs

```diff
@@ -413,9 +413,7 @@ impl Consumer {
         let pre_results = vec![Ok(()); txs.len()];
         let check_results =
             bank.check_transactions(txs, &pre_results, MAX_PROCESSING_AGE, &mut error_counters);
-        let check_results = check_results
-            .into_iter()
-            .map(|(result, _nonce, _lamports)| result);
+        let check_results = check_results.into_iter().map(|result| result.map(|_| ()));
```

Comment:
This construction is awkward. Are you mapping result to `()`?

Reply:
I'm mapping from `Result<CheckedTransactionDetails>` to `Result<()>`